.gitignore

```diff
@@ -4,3 +4,4 @@ leaderboard.dat
 *.pyc
 target
 .env
+/modules/__pycache__/
```

Comment:
Wouldn't all the files in `__pycache__` be also caught by the `*.pyc` rule?

Reply:
It was either a glitch or something i sent a screenshot on discord